modules/appeals_api/app/controllers/appeals_api/supplemental_claims/v0/supplemental_claims/evidence_submissions_controller.rb

```diff
@@ -1,12 +1,64 @@
 # frozen_string_literal: true
 
 module AppealsApi::SupplementalClaims::V0::SupplementalClaims
-  # rubocop:disable Layout/LineLength
-  class EvidenceSubmissionsController < AppealsApi::V2::DecisionReviews::SupplementalClaims::EvidenceSubmissionsController
-    # rubocop:enable Layout/LineLength
+  class EvidenceSubmissionsController < AppealsApi::ApplicationController
```

Comment:
I used this opportunity to break the inheritance relationship to decision reviews in this controller, since it seemed possible within the 500 line limit